release/README.md

```diff
@@ -37,52 +37,55 @@ https://semver.org/
     **NOTE**: The release action will fail since we haven't tagged the release commit.
     We will do that after QA.
 
-1. Run QA checks on `branch_name` and commit any fixes found during QA into `release-X.Y`.
+1. Create a `QA-X.Y` branch and run QA checks on this branch.
+
+    ```bash
+    git checkout release-X.Y
+    git checkout -b QA-X.Y
+    git push -u origin QA-X.Y
+    ```
 
     **NOTE**: All changes made in QA should be added to `CHANGELOG.md` and **NOT** `WIP-CHANGELOG.md`.
+    Also, make sure to not merge any fixes into `release-X.Y` on this step.
 
-1. Run the release script.
+1. When the QA is finished, create the release tag.
 
     ```bash
-    ./release/release.sh X.Y.0
+    git tag vX.Y.0
     ```
 
-    The release script will:
-    * Amend last commit and prepend message `Release vX.Y.Z`
-    * Create a tag named `vX.Y.Z`
-
 1. Push the tagged commit, create a PR against the release branch and request a review.
 
     ```bash
-    git push --atomic origin branch_name vX.Y.Z
+    git push --atomic origin QA-X.Y vX.Y.0
```

Comment:
Not sure what the difference is, but I used `git push --tags` when I created the 0.12 tag. Would that be better, worse or does it not matter?

Reply:
I think this is a leftover from previous release flow. I think this is because we pushed directly to the release branch and it would start the release workflow directly and we wanted to be sure that the tag and the commit was pushed.
So I guess we can change to `git push --tags QA-X.Y` now